wpilibNewCommands/src/test/native/cpp/frc2/command/CommandSendableButtonTest.cpp

```diff
@@ -49,7 +49,7 @@ TEST_F(CommandSendableButtonTest, trueAndNotScheduledSchedules) {
 
 TEST_F(CommandSendableButtonTest, trueAndScheduledNoOp) {
   // Scheduled and true -> no-op
-  m_command->Schedule();
+  frc2::CommandScheduler::GetInstance().Schedule(m_command);
```

Comment:
`m_command` is an optional which needs to be unwrapped here:

```suggestion
  frc2::CommandScheduler::GetInstance().Schedule(*m_command);
```
Or:

```suggestion
  frc2::CommandScheduler::GetInstance().Schedule(m_command.value());
```